### PR TITLE
feat : added badges

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

**Review by tonymike99 — APPROVED** [review]
Great work!

**Review by humzahasan — COMMENTED** [review]
Looks good! Although would advise to have sm, md , lg for your class name as compared to s,m,l etc. Cause it might be more explanatory.

File: components/badge/src/css/badge-icon.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+.header-img {
+  font-size: 5rem;
+}
+
+.icon-badge-text {
+  position: absolute;
+  font-size: 1.2rem;
+  background-color: var(--offline);
```
Comment (iRohitGaur) [inline components/badge/src/css/badge-icon.css:30]: suggestion: give variable names in uppercase for better visibility

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**